From 43cf8e4d4ec982d253150546b1dfe5b6265bf311 Mon Sep 17 00:00:00 2001
From: Katherine Lu <katherine.lu@freescale.com>
Date: Mon, 2 Mar 2009 15:19:26 +0800
Subject: [PATCH] ENGR00108706 Fix _ipu_dp_set_csc_coefficients can not set the right csc value

Fix _ipu_dp_set_csc_coefficients can not set the right csc value

Signed-off-by: Katherine Lu <katherine.lu@freescale.com>
---
 drivers/mxc/ipu3/ipu_disp.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mxc/ipu3/ipu_disp.c b/drivers/mxc/ipu3/ipu_disp.c
index 979bfed..888c740 100644
--- a/drivers/mxc/ipu3/ipu_disp.c
+++ b/drivers/mxc/ipu3/ipu_disp.c
@@ -684,7 +684,7 @@ void _ipu_dp_set_csc_coefficients(ipu_channel_t channel, int32_t param[][3])
 		return;
 
 	dp_csc_param.mode = -1;
-	dp_csc_param.coeff = &param;
+	dp_csc_param.coeff = param;
 	__ipu_dp_csc_setup(dp, dp_csc_param, true);
 }
 
-- 
1.5.4.4

